src/test/java/com/frameworkium/integration/ai/capture/api/tests/sutnames/SoftwareUnderTestNamesTests.java

```diff
@@ -0,0 +1,62 @@
+package com.frameworkium.integration.ai.capture.api.tests.sutnames;
+
+import com.frameworkium.core.api.tests.BaseTest;
+import com.frameworkium.integration.ai.capture.api.dto.executions.Execution;
+import com.frameworkium.integration.ai.capture.api.dto.sut.Names;
+import com.frameworkium.integration.ai.capture.api.service.executions.ExecutionService;
+import com.frameworkium.integration.ai.capture.api.service.sut.GetSutNamesService;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static com.google.common.truth.Truth.assertThat;
+import static java.util.UUID.randomUUID;
+
+public class SoftwareUnderTestNamesTests extends BaseTest {
+
+    private String expectedSutName;
```

Comment:
could this be `private static final` and just set statically rather than in `setUp`?